digitalist-se/next_path_alias
# Issue #2 Save next sites also when manual path instead of pathauto

# Feature / Fix

**User story:** https://github.com/digitalist-se/next_path_alias/issues/2

**Description:**

* Remove our previous override of `pathauto.alias_storage_helper` service and replace with:
* Set context on node presave.
* Force path_alias save on node post-save.
* Set sites from context on path_alias pre save.

# Testing

1. Repeat tests from https://github.com/digitalist-se/next_path_alias/pull/3 . All pass !!
2. Install a local copy of Forterro Headless.
3. Visit URL aliases page at `/admin/config/search/path` and filter the common alias `/blog`.
4. Configure local frontend app to test that each /blog page is displayed correctly.
5. Create yet another content with path `/blog` on Forterro, configure frontend, Your new page is shown.
6. Edit your page, set other sites. If /blog already exist in that site, you will see an error message, if it doesn't, you won't.
